### PR TITLE
Update migrating-from-azure-pipelines-to-github-actions.md

### PR DESCRIPTION
**Pull request by tedescomicchidev**
Mentioning the ability to concatenate the stages similar to Azure Pipelines when leveraging the `workflow_run` webhook event trigger.

### Why:

Closes [issue link]

### Check off the following:

- [x] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@tedescomicchidev 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Review by Zringgo77 — APPROVED** [review]

**Review by Zringgo77 — DISMISSED** [review]
Give me win 2000000

File: content/actions/learn-github-actions/migrating-from-azure-pipelines-to-github-actions.md
Comment (lukki25) [inline content/actions/learn-github-actions/migrating-from-azure-pipelines-to-github-actions.md:40]: Usługa Azure Pipelines obsługuje _stages_ zdefiniowane w pliku YAML, który może służyć do tworzenia przepływów pracy wdrażania. {% danych variables.product.prodname_actions %} wymaga oddzielenia etapów na oddzielne pliki przepływu pracy YAML.

Comment (tedescomicchidev) [reply content/actions/learn-github-actions/migrating-from-azure-pipelines-to-github-actions.md:40]: Hi - could you please translate your feedback in english? Thanks

**Review by lukki25 — APPROVED** [review]
Usługa Azure Pipelines obsługuje _stages_ zdefiniowane w pliku YAML, który może służyć do tworzenia przepływów pracy wdrażania. {% danych variables.product.prodname_actions %} wymaga oddzielenia etapów na oddzielne pliki przepływu pracy YAML.

File: content/actions/learn-github-actions/migrating-from-azure-pipelines-to-github-actions.md
Comment (Sondiez) [reply content/actions/learn-github-actions/migrating-from-azure-pipelines-to-github-actions.md:40]: Eveything looks great well done ✅

**Review by Sondiez — DISMISSED** [review]

File: content/actions/learn-github-actions/migrating-from-azure-pipelines-to-github-actions.md
Comment (skedwards88) [inline content/actions/learn-github-actions/migrating-from-azure-pipelines-to-github-actions.md:40]: minor wording change to clarify a bit
```suggestion
- Azure Pipelines supports _stages_ defined in the YAML file, which can be used to create deployment workflows. {% data variables.product.prodname_actions %} requires you to separate stages into separate YAML workflow files. You can use the `workflow_run` webhook event trigger to run the {% data variables.product.prodname_actions %} workflows one after another, similar to Azure Pipelines.
```

**Review by skedwards88 — COMMENTED** [review]
🎉 Great addition! Thanks for sharing this! I will make one minor wording change, then get this merged down for you.

**Review by skedwards88 — APPROVED** [review]

**Comment by skedwards88** [comment]
@tedescomicchidev It looks like I don't have permission to resolve the merge conflict. I think if you enable the checkbox to allow maintainer edits, then I will be able to resolve this. (https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork ) Alternatively, could you resolve the merge conflict?

**Comment by ramyaparimi** [comment]
@tedescomicchidev The PR has some unresolved conflicts which is blocking the merge 💛  Could you please try resolving the conflict by merging into your working branch from main? Once the conflicts are resolved, I will be able to merge this PR ✨ 

Thanks so much for your time in contributing to the GitHub docs 💖

**Comment by korgpa** [comment]
Please I need my photos that have been deleted from my iPhone photo album on Apple ID hzip22@icloud.com In the year 2020 during the period from 1/2/2020 to 1/5/2020

File: content/actions/learn-github-actions/migrating-from-azure-pipelines-to-github-actions.md
Comment (Thuyduo) [reply content/actions/learn-github-actions/migrating-from-azure-pipelines-to-github-actions.md:40]: Xin chào

**Pull request closed without merging**
